mne/viz/evoked.py

```diff
@@ -393,7 +394,8 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
 def plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
                 ylim=None, xlim='tight', proj=False, hline=None, units=None,
                 scalings=None, titles=None, axes=None, gfp=False,
-                window_title=None, spatial_colors=False):
+                window_title=None, spatial_colors=False,
+                set_tight_layout=True):
```

Comment:
+1 I don't like this set_tight_layout param.

we've always managed to handle it in the past across mpl backends